app/tests/integration/nimbus/pages/browser/__init__.py

```diff
@@ -42,7 +42,7 @@ def wait_for_pref_flip(self, pref, pref_value):
                 assert pref_value in [
                     element.text for element in elements
                 ], "Pref not found"
-            except (Exception, AssertionError) as e:
+            except Exception as e:
```

Comment:
I guess an ```AssertionError``` is just a fancy ```Exception```?

Reply:
All exception subclass `Exception` so here you're saying 'match all exceptions that are either a subclass of Exception (which matches all exceptions), or AssertionError', so the assertion error part there is redundant since everything will match the first type.